examples/compress.rs

```diff
@@ -1,122 +0,0 @@
-//! An example of offloading work to a thread pool instead of doing work on the
```

Comment:
Why do you want to delete it?

Reply:
Because all the examples use a thread pool now!